test/container_test.py

```diff
@@ -111,7 +217,7 @@ def test_set_volume_unit_conv(self):
         c.well(1).set_volume(".1:milliliter")
         self.assertTrue(c.well(1).volume == Unit(100, "microliter"))
         with self.assertRaises(ValueError):
-            c.well(2).set_volume("1:liter")
+            c.well(2).set_volume("1:milliliter")
```

Comment:
I think the test wasn't quite right; the volume check in `set_volume` was appearing as skipped in coverage despite this test because it was never being exercised. set_volume _was_ failing with a ValueError, but the ValueError was being emitted from `convert_to_ul` because it doesn't know what a liter is (it understands [nl, µl, and ml](https://github.com/autoprotocol/autoprotocol-python/blob/master/autoprotocol/util.py#L16)).  If you move the "liter" test outside of the `assertRaises` block you'll see that the traceback isn't from the check in `set_volume`.